ceiling-tests/v2/README.md

```diff
@@ -0,0 +1,606 @@
+| **L.220** | `m7i.4xlarge`  | 2 x`c7i.large` | 2 x `db.r7g.large` | No             | **231**             | DB             |
+| **L.222** | `m7i.4xlarge`  | 2 x`c7i.large` | 2 x `db.r7g.large` | yes            | **2221**            | DB             |
+
+#### Family XL
+
+|            | **Proxy type** | **App nodes**    | **DB nodes**        | **ES** | **Supported users** | **Bottleneck** |
+|------------|----------------|------------------|---------------------|--------|---------------------|----------------|
+| **XL.220** | `m7i.4xlarge`  | 2 x `c7i.xlarge` | 2 x `db.r7g.xlarge` | No     | **17206**           | CPU            |
+| **XL.222** | `m7i.4xlarge`  | 2 x `c7i.xlarge` | 2 x `db.r7g.xlarge` | Yes    | **17128**           | CPU/RAM        |
+
+#### Family 2XL
+
+|             | **Proxy type** | **App nodes**     | **DB nodes**         | **ES** | **Supported users** | **Bottleneck** |
+|-------------|----------------|-------------------|----------------------|--------|---------------------|----------------|
+| **2XL.220** | `m7i.4xlarge`  | 2 x `c7i.2xlarge` | 2 x `db.r7g.2xlarge` | No     | **29504**           | CPU            |
+| **2XL.222** | `m7i.4xlarge`  | 2 x `c7i.2xlarge` | 2 x `db.r7g.2xlarge` | Yes    | **30661**           | CPU            |
+| **2XL.330** | `m7i.4xlarge`  | 3 x `c7i.2xlarge` | 3 x `db.r7g.2xlarge` | No     | **43371**           | CPU            |
+| **2XL.440** | `m7i.4xlarge`  | 4 x `c7i.2xlarge` | 4 x `db.r7g.2xlarge` | No     | **51572**           | CPU            |
+| **2XL.550** | `m7i.4xlarge`  | 5 x `c7i.2xlarge` | 5 x `db.r7g.2xlarge` | No     | **61461**           | CPU            |
+| **2XL.552** | `m7i.4xlarge`  | 5 x `c7i.2xlarge` | 5 x `db.r7g.2xlarge` | Yes    | **58851**           | CPU (ES CPU)   |
+
+#### Family 4XL
+
+|             | **Proxy type** | **App nodes**     | **DB nodes**         | **ES** | **Supported users** | **Bottleneck**                 |
+|-------------|----------------|-------------------|----------------------|--------|---------------------|--------------------------------|
+| **4XL.440** | `m6in.8xlarge` | 4 x `c7i.4xlarge` | 4 x `db.r7g.4xlarge` | No     | **80573**           | CPU                            |
+| **4XL.442** | `m6in.8xlarge` | 4 x `c7i.4xlarge` | 4 x `db.r7g.4xlarge` | Yes    | **80532**           | CPU (ES CPU)                   |
+| **4XL.550** | `m6in.8xlarge` | 5 x `c7i.4xlarge` | 5 x `db.r7g.4xlarge` | No     | **93808**           | CPU                            |
+| **4XL.552** | `m6in.8xlarge` | 5 x `c7i.4xlarge` | 5 x `db.r7g.4xlarge` | Yes    | **91571**           | CPU (network, cluster, ES CPU) |
+| **4XL.662** | `m6in.8xlarge` | 6 x `c7i.4xlarge` | 6 x `db.r7g.4xlarge` | Yes    | **100926**          | CPU (network, cluster, ES CPU) |
+
+
+### Insights
+
+Apart from the raw numbers in the previous table, running these tests revealed some key points:
+
+- The main bottleneck we see across most of the tests is CPU.
```

Comment:
I'd clarify which CPU we are talking about.

Reply:
Good point, done in https://github.com/mattermost/performance-reports/pull/22/commits/f43d0e41ffb270757875997d5d396b28701f6a75